build(aio): upgrade to cli 1.6.3

IgorMinar opened the pull request.

IgorMinar committed:
build(aio): upgrade to @angular/cli@1.6.3

This change introduces a size regression of 9kb for main.js :-(

I filed an issue for this: https://github.com/angular/angular-cli/issues/9108

-rw-r--r--  1 iminar  eng   72546 Jan  5 19:27 dist.cli-1.6.3/0.86a7a08f7866e6cdc36f.chunk.js
-rw-r--r--  1 iminar  eng   14893 Jan  5 19:27 dist.cli-1.6.3/4.c719ac5645940382cdce.chunk.js
-rw-r--r--  1 iminar  eng    1558 Jan  5 19:27 dist.cli-1.6.3/inline.7722895d8c844f710bcd.bundle.js
-rw-r--r--  1 iminar  eng  466484 Jan  5 19:27 dist.cli-1.6.3/main.faff0e2da95443f759f5.bundle.js
-rw-r--r--  1 iminar  eng   40363 Jan  5 19:27 dist.cli-1.6.3/polyfills.87edf5d695f14a29bf91.bundle.js
-rw-r--r--  1 iminar  eng   54001 Jan  5 19:27 dist.cli-1.6.3/worker-basic.min.js

mary-poppins commented:
You can preview d43e8b6 at https://pr21350-d43e8b6.ngbuilds.io/.

FrozenPandaz commented on aio/yarn.lock:
```diff
@@ -8440,15 +8445,19 @@ typescript@2.4:
+  resolved "https://registry.yarnpkg.com/typescript/-/typescript-2.6.2.tgz#3c5b6fd7f6de0914269027f03c0946758f7673a4"
+
+uglify-es@^3.3.4:
+  version "3.3.4"
```
For some reason this did not move to 3.3.5

IgorMinar replied:
3.3.5 was released after this PR was created. I updated the lock file to update to 3.3.5 now.

IgorMinar committed:
build(aio): upgrade to uglify-es@3.3.5

This should fix the size regression spotted via the previous commit.

-rw-r--r--  1 iminar  eng   72498 Jan  8 00:03 dist/0.86a7a08f7866e6cdc36f.chunk.js
-rw-r--r--  1 iminar  eng   14872 Jan  8 00:03 dist/4.c719ac5645940382cdce.chunk.js
-rw-r--r--  1 iminar  eng    1558 Jan  8 00:03 dist/inline.7722895d8c844f710bcd.bundle.js
-rw-r--r--  1 iminar  eng  453905 Jan  8 00:03 dist/main.faff0e2da95443f759f5.bundle.js
-rw-r--r--  1 iminar  eng   40264 Jan  8 00:03 dist/polyfills.87edf5d695f14a29bf91.bundle.js
-rw-r--r--  1 iminar  eng   54001 Jan  8 00:03 dist/worker-basic.min.js

mary-poppins commented:
You can preview ec38dba at https://pr21350-ec38dba.ngbuilds.io/.

mary-poppins commented:
You can preview d2becd1 at https://pr21350-d2becd1.ngbuilds.io/.

petebacondarwin commented:
The failure in Travis was because the builds were actually "smaller" than the expected size (by more than the prescribed 1%).

gkalpak commented:
The sizes still need to be updated (decreased) afaict.

mary-poppins commented:
You can preview f0762aa at https://pr21350-f0762aa.ngbuilds.io/.

mary-poppins commented:
You can preview f91dc81 at https://pr21350-f91dc81.ngbuilds.io/.
